_sass/vrsite/assignment.scss

```diff
@@ -0,0 +1,24 @@
+.mp-content {
+	h2 button,h3 button {
+		all: inherit;
+		display: inline;
+	}
+
+	h2 button:focus svg, h3 button:focus svg {
+		outline: 2px solid;
+	}
+	  
+	button svg {
```

Comment:
nitpicking here but adding
`cursor: pointer;`
to the button would make UX nicer

Reply:
Also consider adding
`transform: translate(0, 0.15em);`
to make the +/- button more aligned with the heading (right now they're slightly higher than the center of the heading)

Reply:
Did we think that we shouldn't make the Submission Instructions section collapsible, or is it not collapsible only because it's h2?

Reply:
Re:submission instructions - the sections are only collapsible in `.mp-content`, which doesn't include Submission Instructions. I'll work on separating this feature out into something more modular so that it can be applied to other sections of the site, like the syllabus.

Reply:
That'd be great. The project gallery could also make use of it